src/components/flash/flash.style.js

```diff
@@ -0,0 +1,136 @@
+import styled, { css } from 'styled-components';
+
+const colors = {
+  warning: '#FF7D00',
+  default: '#335B6D',
+  error: '#c7384f',
+  info: '#1573E6',
+  new: '#663399',
+  success: '#50B848',
+  help: '#FFAB00',
+  maintenance: '#FF7D00'
+};
```

Comment:
shouldn't these colors be defined in a theme config?

Reply:
This is classic style